include/z64environment.h

```diff
@@ -132,17 +132,34 @@ typedef struct {
     /* 0x06 */ u8 skybox2Index;
 } TimeBasedSkyboxEntry; // size = 0x8
 
+// Get blend rate from `EnvLightSettings.blendRateAndFogNear` in 0-255 range
+#define ENV_LIGHT_SETTINGS_BLEND_RATE_U8(blendRateAndFogNear) (((blendRateAndFogNear) >> 10) * 4)
+#define ENV_LIGHT_SETTINGS_FOG_NEAR(blendRateAndFogNear) ((blendRateAndFogNear) & 0x3FF)
+
 typedef struct {
     /* 0x00 */ u8 ambientColor[3];
     /* 0x03 */ s8 light1Dir[3];
     /* 0x06 */ u8 light1Color[3];
     /* 0x09 */ s8 light2Dir[3];
     /* 0x0C */ u8 light2Color[3];
     /* 0x0F */ u8 fogColor[3];
-    /* 0x12 */ s16 fogNear;
-    /* 0x14 */ s16 fogFar;
+    /* 0x12 */ s16 blendRateAndFogNear;
+    /* 0x14 */ s16 zFar;
 } EnvLightSettings; // size = 0x16
 
+typedef EnvLightSettings LightSettings;
```

Comment:
It seems appropriate to add a comment explaining what this is for, suggestion:
```suggestion
// For ZAPD compatibility, TODO remove with a ZAPD update
typedef EnvLightSettings LightSettings;
```